lib/tbd/psi.rb

```diff
@@ -2259,15 +2333,17 @@ def process(model = nil, argh = {})
         end
 
         edge[:surfaces].keys.each do |i|
-          break     if is[:rimjoist   ] || is[:balcony        ] ||
-                       is[:balconysill] || is[:balconydoorsill]
+          break     if is[:ceiling        ]
```

Comment:
If an edge has been previously tagged as a "ceiling", don't bother checking for "rimjoist/balcony" variants.